sdk/eventhub/perf-tests/event-hubs/test/send.spec.ts

```diff
@@ -25,14 +25,14 @@ export class SendTest extends PerfTest<SendTestOptions> {
       required: true,
       description: "Size in bytes",
       shortName: "sz",
-      longName: "size",
+      longName: "body-size",
```

Comment:
```suggestion
      longName: "event-size",
```

How about we use `event-size` to align with both the "subscribe" JS test and Python?